cli/find.ts

```diff
@@ -0,0 +1,41 @@
+import { getFrequencyAPI } from "./services/connect.js";
+import { schemas } from "../dsnp/index.js";
+
+const find = async () => {
+  const api = await getFrequencyAPI();
+
+  console.log("\n## DSNP Schema Information");
+
+  for (const schemaEntry of schemas.entries()) {
+    const schemaString = JSON.stringify(schemaEntry[1].model);
+    const schemaVersionResult = (await api.rpc.schemas.getVersions("dsnp." + schemaEntry[0])).unwrap();
```

Comment:
nit: you might also want to print all the schema names and their versions for "dsnp" namespace.

you can use the following:
`const allDsnp = (await api.rpc.schemas.getVersions("dsnp")).unwrap();`

Reply:
Oh, that's simpler. Will do.